packages/@react-aria/actiongroup/src/useActionGroup.ts

```diff
@@ -41,23 +39,36 @@ export function useActionGroup<T>(props: AriaActionGroupProps<T>, state: ListSta
   }
 
   let {direction} = useLocale();
-  // eslint-disable-next-line arrow-body-style
-  let keyboardDelegate = useMemo(() => {
-    return new ActionGroupKeyboardDelegate(state.collection, direction, orientation, state.disabledKeys);
```

Comment:
It's only gotten rid of here because it's not using `useSelectableCollection` anymore. And that's because we sometimes don't display the entire collection now (when collapsed), so we need to rely on the DOM for keyboard navigation rather than the collection itself. I don't think we should apply this generally.